engine/src/main/java/org/destinationsol/game/ship/FarShip.java

```diff
@@ -189,4 +190,24 @@ public float getMoney() {
     public ItemContainer getIc() {
         return container;
     }
+
+    /**
+     * Each FarShip could be a mercenary. Each mercenary is definitely a FarShip .
+     * This method sets the associated MercItem.
```

Comment:
Not really needed considering the signature of the method. Better to rephrase as `Sets this ship's MercItem`